hi-ml-cpath/src/health_cpath/utils/plots_utils.py

```diff
@@ -216,6 +234,28 @@ def get_list_from_results_dict(results: ResultsType, results_key: ResultsKey) ->
     return [i.item() if isinstance(i, Tensor) else i for i in results[results_key]]
 
 
+def get_stratified_outputs(true_labels: List[Any], scores: List[Any],
+                           stratify_metadata: List[Any]) -> Dict[str, List[Any]]:
+    """
+    Get stratified true labels and predictions given metadata, from all true and prediction labels
+    to plot stratified curves.
+    :param true_labels: list of true labels.
+    :param scores: list of prediction scores.
```

Comment:
Empty line before :param starts
```suggestion

    :param scores: list of prediction scores.
```

Reply:
thanks, done